server/keyshare/keyshareserver/server.go

```diff
@@ -122,6 +122,14 @@ func (s *Server) Handler() http.Handler {
 		router.Route("/api/v1", func(r chi.Router) {
 			s.routeHandler(r)
 		})
+
+		router.Route("/api/v2", func(r chi.Router) {
+			// Keyshare sessions with provably secure keyshare protocol
+			r.Use(s.userMiddleware)
+			r.Use(s.authorizationMiddleware)
+			r.Post("/prove/getPs", s.handlePs)
+			r.Post("/prove/getCommitments", s.handleCommitmentsV2)
+		})
```

Comment:
So adding the `getResponse` here is still a TODO?

Reply:
yep, i've hardly added any todo's since we try to make the PRs small. the story mapping gives an idea of what is yet to be done.